Naveen-gali/Yelp-App
# Category Section

### What does this PR do?

- Adds Category Section

### Related Issues

- closes #45 
- closes #50 

### How to test

- Clone the Repository.
- Switch to feat/categorySection branch.
- Do yarn to install the dependencies.
- Do yarn run **build-stage-ios** for ios or **yarn run build-stage-android** for android.
- You Can see the Grid layout of the Categories on the screen

### Screenshots


### Android Screen Recording.

https://github.com/Naveen-gali/Yelp-App/assets/122969303/efbe1710-521e-4c1f-8104-1715a126531b



### Ios Screen Recording.
https://github.com/Naveen-gali/Yelp-App/assets/122969303/7437a0be-da4a-45ae-9812-4a9807a87abb 

### Checklist
- [x] I have addressed all the requirements.
- [x] I have self reviewed the PR.

### Additional Notes 

- [FlatList Render Item Arrow Functions](https://stackoverflow.com/a/69732518)
